crates/stellar/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub mod address_conv;
+pub mod balance_conv;
+pub mod currency;
+pub mod currency_conv;
+pub mod horizon;
+
+pub use substrate_stellar_sdk as substrate_sdk;
```

Comment:
@gonzamontiel  here I made it public, so that other crates which uses the stellar pallet, won't need to add
`substrate_stellar_sdk` as a dependency in their `cargo.toml`.